Formula/c/clusterctl.rb

```diff
@@ -1,8 +1,9 @@
 class Clusterctl < Formula
   desc "Home for the Cluster Management API work, a subproject of sig-cluster-lifecycle"
   homepage "https://cluster-api.sigs.k8s.io"
-  url "https://github.com/kubernetes-sigs/cluster-api/archive/refs/tags/v1.7.4.tar.gz"
-  sha256 "c0acb1626a3948107db8131aff01900f9f097fc0b854ef5792fcd91dd324e4f6"
+  url "https://github.com/kubernetes-sigs/cluster-api.git",
+      tag:      "v1.8.0",
+      revision: "69bbf3530777a81867328c9379b41f814b008cdb"
```

Comment:
why do we do this?

Reply:
according to this guide: https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md#to-submit-a-version-upgrade-for-the-foo-formula 
It should be allowed to use tag and revision. But I can change it to sha256 if that is better

Reply:
changed it to url and sha

Reply:
yeah, we'd prefer tarball/sha whenever possible, but we did run into some go build with helm before https://github.com/Homebrew/homebrew-core/pull/177094